openhtf/io/frontend/__main__.py

```diff
@@ -96,47 +96,41 @@ class StationStore(threading.Thread):
   """
   def __init__(self):
     super(StationStore, self).__init__()
-    self._death_event = threading.Event()
+    self._stop_event = threading.Event()
     self.hostname = socket.gethostname()
     self.stations = collections.defaultdict(Station)
```

Comment:
this is probably a complaint of mine about the Python 'use tuples as structs' paradigm, but i don't like to see things like `(host, port)` tuples that aren't namedtuples, because it relies on an implied mapping of index to semantic meaning of respective value, that's only found in comments/docstrings.

the python socket library should provide that namedtuple, since its methods use it as inputs, but it doesn't and that makes me sad.  if you feel like it, you could make a namedtuple to use instead, but i won't give you too hard a time about it since it's really a python socket interface thing.